galsim/config/stamp.py

```diff
@@ -717,19 +717,26 @@ def locateStamp(self, config, base, xsize, ysize, image_pos, world_pos, logger):
         elif world_pos is not None and image_pos is None:
             image_pos = wcs.toImage(world_pos)
 
+        # If the world_pos is a CelestialCoord, then we also call it sky_pos.
+        # If the world_pos is not celestial, then the user may optionally define a sky_pos
+        # value, which gets saved as base['sky_pos'].  This may be useful for things that need
+        # to know where in the sky the pointing is, even if the WCS is not a CelestialWCS.
```

Comment:
The comment here doesn't quite match the logic of the code. Not sure if it has to but worth thinking about what the correct behavior is.

Reply:
Good point.  A user-defined sky_pos takes precedence.  Added "or the user wants to override it for some reason,"